code/__DEFINES/access.dm

```diff
@@ -64,6 +64,9 @@
 #define ACCESS_NETWORK 67 //! NTnet diagnostics/monitoring software
 #define ACCESS_CLONING 68 //! Cloning room and clone pod ejection
 
+/// Room and launching.
+#define ACCESS_AUX_BASE 72
```

Comment:
14 would be a pretty good number to choose, since that slot is empty and it is close to engineering

Reply:
Don't really want to break from TG's PR, since it would make adding jobs/changing maps annoying if we ever port something from them.

https://tgstation13.org/wiki/Guide_to_door_access

![image](https://user-images.githubusercontent.com/3241376/102260887-8af27100-3ed6-11eb-8263-1356e43cdfe6.png)

Reply:
I agree with the bloons and I think it's better to let it stay as 72 just for further compatibility.

Reply:
enough to convince me